src/rgw/rgw_rados.cc

```diff
@@ -5723,12 +5779,107 @@ int RGWRados::swift_versioning_copy(RGWBucketInfo& bucket_info, RGWRados::Object
                NULL, /* void (*progress_cb)(off_t, void *) */
                NULL); /* void *progress_data */
   if (r == -ECANCELED || r == -ENOENT) { /* has already been overwritten, meaning another rgw process already copied it out */
+  ldout(cct, 20) << "++++ ECANC   --- SWIFT VERSION COPY BEGIN ++++ LINE: " << __LINE__ << dendl;
```

Comment:
Oops, accidentally incorporated this line during `git add -p`. I will rework. Thanks for pointing this out.